### ISSUE TITLE
<title>The consistency of dmtet model generation results is poor

### ISSUE THREAD

**Issue by yudefu** [ashawkey/stable-dreamfusion#225]
### Description

Hello author, when I use text and images to generate 3D models, when I call the DMTET model, the consistency of the generated sample results is relatively poor. I don’t know the reason, can you help me to answer it? Thank you so much!

### Steps to Reproduce

1、python3 main.py -O --image data/fox_rgba.png --text "a fox sitting on the ground" --workspace trial_imagetext_fox --iters 5000
2、python3 main.py --image data/fox_rgba.png --text "a fox sitting on the ground" --workspace trial2_imagetext_fox --iters 10000 --dmtet --init_ckpt trial_imagetext_fox/checkpoints/df.pth

### Expected Behavior

I hope that the rendering result of the fox should have only one head, but the generated result has several fox heads

### Environment

Ubuntu 20.04，pytorch_lightning 1.9.4，cuda：10.2

**Comment by ashawkey** [comment]
@yudefu Hi, providing both image and text is still experimental and I haven't got the time to tune its performance. You may try to adjust the parameters in `main.py` if you really want to use this feature.

**Comment by jiaxinxie97** [comment]
@ashawkey Hi, I am wondering if the image-to-3d.mp4 and text-to-3d.mp4 in the Readme is your results using this repo? For the image-to-3d, I get a very poor result in a simply toy car image without text by using the similar command as the @yudefu. 
python main.py -O --image toy_car_rgba.png --workspace trial_image --iters 5000
python main.py -O --image toy_car_rgba.png --workspace trial_image_dmtet --dmtet --init_ckpt trial_image/checkpoints/df.pth

For the text-to-3d, I also can not get a comparable result with the text "a DSLR photo of a delicious hamburger". the command is 
python main.py --text "a DSLR photo of a delicious hamburger" --workspace trial -0

So I would like to ask if I want to get the results you showed, I can’t use the example command, I should I adjust the paramters manually?

**Comment by ashawkey** [comment]
@jiaxinxie97 Yes, as stated in readme, image-to-3d still fails for many images, you could try other methods if not satisfied. The demo is after DMTet finetuning, you need to run both stages. You could check the examples under `scripts` for reproduction.
